UnitTestFrameworkPkg/ReadMe.md

```diff
@@ -399,6 +399,37 @@ CMOCKA_XML_FILE=<absolute or relative path to output file>
 
 This mode is used by the test running plugin to aggregate the results for CI test status reporting in the web view.
 
+### Code Coverage
+
+Host based Unit Tests will automatically enable coverage data.
+
+For Windows, This is primarily leverage for pipeline builds, but this can be leveraged locally using the
+OpenCppCoverage windows tool, and pycobertura is used to covert this coverage data to a human readable HTML
+file. These tools must be installed to parse code coverage.
+
+- Windows Prerequisite
+  ```bash
+  Download and install https://github.com/OpenCppCoverage/OpenCppCoverage/tags

```

Comment:
i would reference the release page instead of the tag page. 

https://github.com/OpenCppCoverage/OpenCppCoverage/releases

Reply:
Okey, will update it in next patch.